packages/console/src/assets/docs/guides/m2m-general/README.mdx

```diff
@@ -54,12 +57,12 @@ In the API Resource tab, find the API identifier that the app needs to access. I
     <a href="https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Authorization#basic_authentication">
       Basic authentication
     </a>
-    , where username is the App ID, and password is the App Secret.
+    , where username is the App ID, and password is the App Secret. The credential is the username and password combined with a colon, encoded in base64.
```

Comment:
it'll be great if we can have an example here

Reply:
Alright, I guess I can write the code here as an example. And we do calculate the base64 value for users in the code sample below

Reply:
No, I changed my mind. It reads too wordy.